catalog/spatial/spatial-commands/src/main/java/org/codice/ddf/commands/spatial/gazetteer/GazetteerUpdateCommand.java

```diff
@@ -85,41 +102,49 @@ public Object execute() {
     final PrintStream console = System.out;
 
     final ProgressCallback progressCallback =
-        new ProgressCallback() {
-          @Override
-          public void updateProgress(final int progress) {
-            console.printf("\r%d%%", progress);
-            console.flush();
-          }
+        progress -> {
+          console.printf("\r%d%%", progress);
+          console.flush();
+        };
+
+    final FeatureIndexer.IndexCallback featureIndexCallback =
+        count -> {
+          console.printf("\r%d features indexed", count);
+          console.flush();
         };
 
     console.println("Updating...");
 
     try {
-      geoEntryIndexer.updateIndex(resource, geoEntryExtractor, create, progressCallback);
+      if (isResourceGeoJSON()) {
+        featureIndexer.updateIndex(resource, featureExtractor, create, featureIndexCallback);
+      } else {
+        geoEntryIndexer.updateIndex(resource, geoEntryExtractor, create, progressCallback);
+      }
+
       console.println("\nDone.");
-    } catch (GeoEntryExtractionException e) {
-      LOGGER.debug("Error extracting GeoNames data from resource {}", resource, e);
+    } catch (GeoEntryExtractionException | FeatureExtractionException e) {
+      LOGGER.debug("Error extracting data from resource {}", resource, e);
       console.printf(
-          "Could not extract GeoNames data from resource %s.%n"
-              + "Message: %s%n"
-              + "Check the logs for more details.%n",
+          "Could not extract data from resource %s.%n Message: %s%n Check the logs for more details.%n",
           resource, e.getMessage());
-    } catch (GeoEntryIndexingException e) {
-      LOGGER.debug("Error indexing GeoNames data", e);
+    } catch (GeoEntryIndexingException | FeatureIndexingException e) {
+      LOGGER.debug("Error indexing data", e);
       console.printf(
-          "Could not index the GeoNames data.%n"
-              + "Message: %s%n"
-              + "Check the logs for more details.%n",
+          "Could not index the  data.%n Message: %s%n heck the logs for more details.%n",
           e.getMessage());
     } catch (GeoNamesRemoteDownloadException e) {
       LOGGER.debug("Error downloading resource from remote source {}", resource, e);
       console.printf(
-          "Could not download the GeoNames file %s.%n  Message: %s%n"
-              + "Check the logs for more details.%n",
+          "Could not download the GeoNames file %s.%n  Message: %s%n Check the logs for more details.%n",
```

Comment:
❗️ the problem which each of these console messages is that the additional information in the logs is likely at a DEBUG level and won't be in the logs by default.  I think we should either log the error messages and stack traces at INFO or higher or tell the user to re-run the command after turning up log levels on the proper package(s)

Reply:
changed to info